src/wp-includes/class-wp-plugin-dependencies.php

```diff
@@ -359,10 +359,16 @@ public static function get_dependency_data( $slug ) {
 	 */
 	public static function display_admin_notice_for_unmet_dependencies() {
 		if ( in_array( false, self::get_dependency_filepaths(), true ) ) {
+			$plugins_url = is_multisite() ? network_admin_url( 'plugins' ) : admin_url( 'plugins' );
 			wp_admin_notice(
-				__( 'There are additional plugin dependencies that must be installed.' ),
+				sprintf(
+					/* translators: 1: The start of a link to the plugins page, 2: The end of the link. */
+					__( 'Some required plugins are missing. Please %1$sreview your plugins%2$s for plugins that cannot be activated.' ),
```

Comment:
I agree with the feedback that plugins appears too many times.

I'd love if we had a more concrete step for the user. "Review your plugins" is not the greatest direction. What are they looking for?

Active and inactive plugins have different background colors, and when a site is in recovery mode, there is a third styling to call attention to the problematic rows.

![Screenshot 2024-02-12 at 10 12 41 AM](https://github.com/WordPress/wordpress-develop/assets/359867/01a85051-4431-442c-9c05-931c82b9ab13)

What about styling plugins with unmet dependencies differently?

Another option could be making the greyed out `Active` link was changed to say `Missing requirements`?

What would be really nice is a link to a list of all plugins missing dependencies. I opened a suggestion for that in [Trac-60507](https://core.trac.wordpress.org/ticket/60507). Then it could read "View plugins with missing requirements" and link to that filtered view.

Reply:
I am far from an accessibility expert but I thought color change wasn't supposed to be a great differentiator.